neo/rawio/medrawio.py

```diff
@@ -0,0 +1,238 @@
+"""
+Class for reading MED (Multiscale Electrophysiology Data) Format.
+    
+Uses the dhn-med-py python package, created by Dark Horse Neuro, Inc.
+
+Authors: Matt Stead, Dan Crepeau, Casey Stengel
+"""
+
+from .baserawio import (BaseRawIO, _signal_channel_dtype, _signal_stream_dtype,
+                _spike_channel_dtype, _event_channel_dtype)
+
+import numpy as np
+
+
+class MedRawIO(BaseRawIO):
+    """
+    Class for reading MED (Multiscale Electrophysiology Data) Format.
+    
+    Uses the dhn-med-py MED python package (version >= 0.9.5), created by
+    Dark Horse Neuro, Inc.
+    
+    Currently reads the entire MED session into one block/segment/stream.  If
+    channels have differing sampling frequencies, all frequencies are upsampled
+    to the highest frequency prior to returning the chunk.
+    
+    Timestamps generated are referenced to the beginning of the session,
+    with the beginning of the session being timestamp zero.  Gaps in the data
+    are ignored for the purposes of timestamping.
```

Comment:
Ignoring gaps of arbitrary sizes can lead to very wrong timestamps without the user having a chance to notice this. If you want to permit loading data sets with small variations in the timestamps of a continuous signal I would propose to introduce a flag for this, e.g. `ignore-timestamp-gaps`. In my opinion if a gap is larger than the sampling rate this discrepancy should not be silently ignored, but either raise an error or be split into multiple segments. @samuelgarcia What do you think about this?